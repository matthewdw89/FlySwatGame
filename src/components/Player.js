import React, { Component } from 'react'

export default class Player extends Component {
 

  render() {
    return (
        <div ref={this.props.playerRef} >
            <img id="player" src='./images/player.png' style={this.props.position} alt="A cartoon drawing of a swatter that has arms and legs"></img>
        </div>
    )
  }
}
