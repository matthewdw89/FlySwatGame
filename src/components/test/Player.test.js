import React from 'react';
import { shallow, mount, render, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Player from '../Player';

configure({ adapter: new Adapter() });

describe('<Player>', () => {

  it('should render correctly', () => {
    const component = shallow(<Player />);
    expect(component).toMatchSnapshot();
  });

});