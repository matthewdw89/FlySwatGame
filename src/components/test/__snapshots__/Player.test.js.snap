// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Player> should render correctly 1`] = `
<div>
  <img
    alt="A cartoon drawing of a swatter that has arms and legs"
    id="player"
    src="./images/player.png"
  />
</div>
`;
